packages/yarnpkg-nm/sources/hoist.ts

```diff
@@ -396,7 +396,12 @@ const hoistTo = (tree: HoisterWorkTree, rootNodePath: Array<HoisterWorkTree>, ro
 
   const hoistIdentMap = getHoistIdentMap(rootNode, preferenceMap);
 
-  const usedDependencies = tree == rootNode ? new Map() : (options.fastLookupPossible ? getZeroRoundUsedDependencies(rootNodePath) :  getUsedDependencies(rootNodePath));
+  const usedDependencies = tree == rootNode ? new Map() :
+    (options.fastLookupPossible
+      ? getZeroRoundUsedDependencies(rootNodePath)
+      :  getUsedDependencies(rootNodePath)
```

Comment:
nit: remove extra space
```suggestion
      : getUsedDependencies(rootNodePath)
```